public/js/user.js

```diff
@@ -334,6 +335,23 @@ apos.define('apostrophe-workflow', {
       });
     };
 
+    self.enableRevert = function() {
+      apos.ui.link('apos-workflow-revert', null, function($el, id) {
+        apos.ui.globalBusy(true);
+        self.api('revert', {id: id}, function (result) {
```

Comment:
Whitespace should be consistent.

We have an `eslintrc` in this repo. If you're going to use an editor that doesn't automatically `eslint`, please run `npm test` or `node ./node_modules/.bin/eslint .` before committing.

Reply:
Whitespace is still not consistent, should be `{ id: id }`

Reply:
done